README.md

```diff
@@ -1,20 +1,80 @@
+![viame logo](http://www.viametoolkit.org/wp-content/uploads/2016/08/viami_logo.png)
+
+## Demo
 
-<img src="http://www.viametoolkit.org/wp-content/uploads/2016/08/viami_logo.png" alt="VIAME Logo" width="200" height="78">
-<br>
-VIAME Web is a web interface for performing data management, video annotation, and running the algorithms stored within
-the VIAME (https://github.com/VIAME/VIAME) repository. When compiled, docker instances for VIAME-Web can either be run as
-local servers, or online in web services.
-<br>
-<br>
-<p align="center">
-<br>
-<nobr>
-<img src="http://www.viametoolkit.org/wp-content/uploads/2019/11/girder-dark-example.png" alt="Dark Girder" width="320" height="200" border="1">
-&nbsp;&nbsp;&nbsp;&nbsp;
-<img src="http://www.viametoolkit.org/wp-content/uploads/2019/11/viame-web-prelim.png" alt="Example Tracks" width="320" height="200" border="1">
-</nobr>
-</p>
 A sample instance of VIAME-Web is running on a public server at https://viame.kitware.com
-<br>
-<br>
```

Comment:
HTML in README docs just increases documentation burden, IMO.  Unless anyone objects, let's use regular markdown.